src/components/SearchToolbar/SearchResult.js

```diff
@@ -0,0 +1,265 @@
+// @flow
```

Comment:
The file name/class is a bit confusing, it contains the whole routing logic - more than just search - , but I wouldn't have guessed.

Reply:
Agree, best would be to inject a function from the outside that creates the necessary `href` for a feature, from <App /> all the way down – what do you think?

Reply:
Made a ticket for this: https://trello.com/c/49XYKKDi/758-side-quest-inject-href-function-for-searchresult-from-outside